.github/workflows/project_automation.yml

```
@@ -27,3 +26,13 @@ jobs:
         env:
           PULL_REQUEST: ${{github.event.pull_request.html_url}}
           GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
+
+  add-to-beta-project:
+    name: Add new issue to the beta Issues Suite project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
```

It would be safer to use specific commit instead of the branch name if we provide the GH token to that action, what do you think?
```suggestion
      - uses: actions/add-to-project@661f5254e8fcead08b054f5bc07b4d7f2af92c44
```

---

we can do that but also we use permission only for this specified thing this action can do.
```yaml
permissions:
  contents: read
  pull-requests: write
  issues: write
  repository-projects: write
 ```

---

so the action can for example delete all our issues, right?